data-gen/gen-bin.py

```diff
@@ -1,37 +1,49 @@
-#!/usr/bin/python
-import sys
-import os
+#!/usr/bin/python3
 import fnmatch
+import os
+
 
 def process(s):
-    print 'Processing %s ...' % s,
-    paths = dict()
-    pkgs = dict()
+    print('Processing %s ...' % s, end=' ')
+    paths = {}
+    pkgs = {}
     f = os.popen('lzma -c -d %s.txt.lzma' % s)
-    g = open(s + '.sql', 'w')
+    g = open(s+'.sql', 'w')
```

Comment:
??? What's the point of this change?

Reply:
Is a PEP8 change, where spaces between operators inside parameters are not recommended

Reply:
OK, I don't want to fight over this silliness, it just seems strange, and I cannot find the chapter and verse of PEP-8 where it would be required. I would probably prefer ``'{}.sql'.format(s)`` anyway.

Reply:
No problem, I can revert this change in a later commit.